README.rst

```diff
@@ -190,6 +190,81 @@ Available languages are stored in a mapping, and so best edited directly in Tuto
 
 Where the first key is the abbreviation of the language to use, "flag" is which flag icon is displayed in the user interface for choosing the language, and "name" is the displayed name for that language. The mapping above shows all of the current languages supported by Superset, but please note that different languages have different levels of completion and support at this time.
 
+Adding custom Row Level Security Filters to Superset
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you add new datasources, tables, or fields to Superset, you may want to add new `row level security filters`_ 
+to restrict access to that data based on people's course roles. To apply custom row level security filters 
+to Superset, you can do so by using the patch `superset-row-level-security`. This patch expects a list 
+of python dictionaries with the following structure:
+
+..  code-block:: yaml
+    
+    superset-row-level-security: |
+        {
+            "schema": "{{ASPECTS_XAPI_DATABASE}}",
+            "table_name": "{{ASPECTS_XAPI_TABLE}}",
```

Comment:
We should be really clear that the schema / table need to exist in the `assets.yaml`, so they can't filter on upstream tables that don't end up getting imported to Superset. I just fought with this for 30 mins trying to figure out why it thought a table didn't exist. Maybe a note about when you see `AssertionError: {schema.table} table doesn't exist yet?` to make sure it's exported to Superset.